index.js

```diff
@@ -0,0 +1,206 @@
+    image: 'img/cards-images/cardfive.jpg',
+    technology: ['ruby on rails', 'css', 'javascript', 'html'],
+  },
+  card6: {
+    id: 5,
+    name: 'Multi-Post Stories Gain+Glory',
+    decription: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam in ligula dictum, volutpat ante id, consectetur leo. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In vel felis faucibus, gravida mi tincidunt, rhoncus risus. Aenean odio velit, mattis eu diam vel, tincidunt ornare mi. In fermentum metus quam, id dignissim libero accumsan in. Ut condimentum ultricies mollis. Aenean at ante ut magna posuere fringilla.',
+    image: 'img/cards-images/cardsix.jpg',
+    technology: ['ruby on rails', 'css', 'javascript', 'html'],
+  },
+};
+
+function popit(cardN) {
+  const modalBody = `
+    <div class='popup-card'>
+        <div class='close-button-container'>
+            <button src='img/Disabled.svg' id='close-button' alt=''>&times</button>
+        </div>
+        <div class='image-container'>
+            <img src='${cardN.image}' class='popup-image' alt='Card Image'>
+        </div>  
+        <div class='popup-title-container'>
+            <p class='popup-title-text'>${cardN.name}</p>
+            <div class='popup-buttons'>
+                <button class='see-button'>
+                    <p class='see-button-text'>See Live</p>
+                    <img src='img/popup-live.svg'  alt='' class='see-live-icon'>
+                </button>
+                <button class='see-button'>
+                    <p class='see-button-text'>See Source</p>
+                    <img src='img/popup-github.svg' alt='' class='see-source-icon'>
+                </button>
+            </div>
+        </div>
+        <ul class='card-tags2'>
+            <li class='card-tag ruby'><p class='tech-item-text'>Ruby on rails</p></li>
+            <li class='card-tag html'><p class='tech-item-text'>html</p></li>
+            <li class='card-tag css'><p class='tech-item-text'>css</p></li>
+            <li class='card-tag javascript'><p class='tech-item-text'>javascript</p></li>
+        </ul>
+        <div class='description-container'>
+            <p class='description'>${cardN.decription}</p>
+        </div>
+        <div class='popup-buttons2'>
+            <button class='see-button'>
+                <p class='see-button-text'>See Live</p>
+                <img src='img/popup-live.svg'  alt='' class='see-live-icon'>
+            </button>
+            <button class='see-button'>
+                <p class='see-button-text'>See Source</p>
+                <img src='img/popup-github.svg' alt='' class='see-source-icon'>
```

Comment:
- Kindly make sure your popup is opaque, not transparent as seen on the Figma design :recycle:
   - You can handle this by giving it a higher opacity than the container, in this particular case it must be the overlay
- Kindly make sure the user can see and access the whole popup including the CTA buttons at the bottom. (for this I suggest using overflow: Scroll and assign overflow: hidden to the body when the popup is active. 
- I also suggest adding a border radius to the popup container and image to stick to the design.

Your design:
![Screenshot from 2023-03-18 12-39-28](https://user-images.githubusercontent.com/97834160/226100683-9f79157d-f1ee-421a-8ef5-88c1623cc9c5.png)
![Screenshot from 2023-03-18 12-38-10](https://user-images.githubusercontent.com/97834160/226100850-9460d2b6-3fe5-4d34-863b-2d5e41b77ea8.png)

Figma:
![Screenshot from 2023-03-18 12-47-23](https://user-images.githubusercontent.com/97834160/226100886-d752d9e4-47a1-4513-a1b4-b2085ffc788f.png)

Reply:
Do you think you can help me understand what the problem is please. I have been struggling with it since last night